mex/pipeline/wikidata.py

```diff
@@ -30,3 +32,34 @@ def extracted_organization_rki(
     )
     load(extracted_organization_rki)
     return extracted_organization_rki[0]
+
+
+def get_organization_merged_id_by_query(
```

Comment:
```suggestion
def get_organization_id_by_query(
```
I would prefer this name, when we're already refactoring. I find `merged` only to be confusing in this function name

Reply:
Naming is based on the convenience function names for ldap in mex-common: https://github.com/robert-koch-institut/mex-common/blob/main/mex/common/ldap/extract.py

I'd rather not change all of these and I like that the function name contains the type of id explicitly, as it could also be the identifier. What do you think?

Reply:
following your suggested convention the function name should then be `get_merged_id_by_query` instead

Reply:
According to our discussion, it is now called `get_merged_organization_id_by_query_with_transform_and_load`